test/specs/simulation.vehicles.spec.js

```diff
@@ -0,0 +1,22 @@
+const { generateRandomVehicles } = require('../../server/simulation/vehicles');
+const { randomDavAddress, randomDroneModel, randomCoords, randomRating, randomMissionsCompleted } = require('../../server/simulation/random');
```

Comment:
There's no need to import (require) every single function from that file, just the one you actually use.

What the `const { myFunc } = require('file.js')` does is actually just yank a single exported object from that file, without requiring all the ones you are not using (this is great for tree shaking dead code later).

If you look at this with an IDE that integrates with ESLint it will pick up on those unused variables and show them to you:

<img width="1024" alt="simulation_vehicles_spec_js_-_missioncontrol_-____dropbox_mysite_dav_missioncontrol_" src="https://user-images.githubusercontent.com/926653/30022870-e4f16d76-9175-11e7-8253-62c2ae5d479b.png">

You can just leave the one you use in there - `randomCoords`